packages/icons/icons.json

```
@@ -373,91 +373,91 @@
         "group": "action",
         "codepoint": 62019
     },
     {
         "displayName": "Maximize",
         "iconName": "maximize",
-        "tags": "arrows, increase, bigger",
+        "tags": "arrows, increase, bigger, expand, opposite diagonal arrows, diagonal bidirectional arrows, double diagonal arrows",
         "group": "action",
         "codepoint": 62012
     },
     {
         "displayName": "Tick",
         "iconName": "tick",
-        "tags": "mark, sign, ok, approved, success",
+        "tags": "mark, sign, ok, approved, success, check, verizon, done, complete, confirm",
         "group": "action",
         "codepoint": 62191
     },
     {
         "displayName": "Cross",
         "iconName": "cross",
-        "tags": "cross mark, fail, delete, no, close, remove",
+        "tags": "cross mark, fail, delete, no, close, remove, not available, wrong/incorrect, cancel, x",
         "group": "action",
         "codepoint": 61801
     },
     {
         "displayName": "Plus",
         "iconName": "plus",
-        "tags": "sign, add, maximize, zoom in",
+        "tags": "sign, add, maximize, zoom in, add or append, enlarge or zoom-in, increase",
         "group": "action",
         "codepoint": 62073
     },
     {
         "displayName": "Minus",
         "iconName": "minus",
-        "tags": "sign, remove, minimize, zoom out",
+        "tags": "sign, remove, minimize, zoom out, line, dash, space, hyphen, emdash, horizonal thin line",
         "group": "action",
         "codepoint": 62020
     },
     {
         "displayName": "Arrow: left",
         "iconName": "arrow-left",
-        "tags": "arrow, direction, left",
+        "tags": "arrow, direction, left, left arrow, backspace, undo, back",
         "group": "interface",
         "codepoint": 61733
     },
     {
         "displayName": "Arrow: right",
         "iconName": "arrow-right",
-        "tags": "arrow, direction, right",
+        "tags": "arrow, direction, right, next, right arrow",
         "group": "interface",
         "codepoint": 61734
     },
     {
         "displayName": "Exchange",
         "iconName": "exchange",
-        "tags": "arrows, direction, exchange, network, swap, transfer, transaction",
+        "tags": "arrows, direction, network, swap, transfer, transaction, repeat, shuffle, cycle",
         "group": "action",
         "codepoint": 61853
     },
     {
         "displayName": "Comparison",
         "iconName": "comparison",
-        "tags": "comparison, analogy, layout, contrast",
+        "tags": "analogy, layout, contrast, compare, check list, split",
         "group": "action",
         "codepoint": 61793
     },
     {
         "displayName": "List",
         "iconName": "list",
-        "tags": "agenda, four lines, table",
+        "tags": "agenda, four lines, table, menu, rows, hamburger, document, writing, justified, hamburger menu, block text, horizontal bars",
```

I don't want to conflate this with "menu", we clearly have two separate icons for lists vs. menus. Same with "align-justify".

```suggestion
        "tags": "agenda, four lines, table, rows, document, writing, block text, horizontal bars",
```